build/targets/KoreBuild.SolutionBuild.targets

```diff
@@ -200,30 +200,6 @@ Executes /t:Pack on all projects matching src/*/*.csproj.
 
 <!--
 ###################################################################
-Target: PackSharedSources
-
-Creates a content files package for all each directory in
-that matches "$(RepositoryRoot)/shared/*.Sources".
-###################################################################
--->
-  <Target Name="PackSharedSources" Condition="Exists('$(SharedSourcesFolder)') OR '@(SharedSourceDirectories)' != ''">
```

Comment:
Not sure I follow - these are for shared src (packages containing src as Content). I don't think source link is meant to be replacement for this.